crypto.py

```diff
@@ -0,0 +1,23 @@
+import nacl.public
+import os
+import pyonionreq.junk
+from const import *
+
+# Copied from https://github.com/oxen-io/session-file-server/blob/dev/fileserver/crypto.py
+
+
+if os.path.exists(PRIVKEY_FILE):
+    with open(PRIVKEY_FILE, "r") as f:
```

Comment:
This isn't going to work as-is: `PRIVKEY_FILE` of the existing server is some pem-encoded junk, which is going to need to be unwrapped and migrated to a raw key value, using a different filename (e.g. `key_x25519`).  There's a script in PySOGS to do the conversion from pem -> raw:

https://github.com/oxen-io/session-pysogs/blob/dev/sogs/key_convert/__main__.py

This is used in the deb upgrade script when upgrading from a pre-pysogs SOGS installation to do the one-time conversion using:
```bash
    if ! [ -e $SOGS/key_x25519 ] && [ -e $SOGS/x25519_private_key.pem ]; then
        echo -n "Importing old SOGS private key"
        cd $SOGS
        su -s /bin/sh _loki -c "python3 -msogs.key_convert -i $SOGS/x25519_private_key.pem -o $SOGS/key_x25519"
    fi
```

Since this is only on a single server (AFAIK) I can't see why we'd need to bother scripting this and can just copy and run the script from pysogs manually (out-of-repo) to create the raw `key_x25519` from a one-time conversion.

Reply:
Apparently this may not be actually pem-encoded, and instead just raw bytes.